torchvision/csrc/cuda/DeformConv_cuda.cu

```diff
@@ -80,12 +80,11 @@
 
 using namespace at;
 
-const int CUDA_NUM_THREADS = 1024;
-const int kMaxGridNum = 65535;
-
+const unsigned int CUDA_NUM_THREADS = 1024;
+const unsigned int kMaxGridNum = at::cuda::getCurrentDeviceProperties()->maxGridSize[0];
```

Comment:
For my understanding, can we make this a `const` in the outer namespace? Doesn't it depend on the machine it's being run on, and thus could change from the machine it was compiled to the machine it's being run on?

Reply:
Yes, I agree that this variable should be assigned at runtime. It is very unlikely that taking into account how implemented [`at::cuda::getCurrentDeviceProperties()`](https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/cuda/CUDAContext.cpp#L41), that `kMaxGridNum` is evaluated at compilation time.

Reply:
Yes, you are right. I'll change it to dynamic evaluation

Reply:
Yes, you are right. I'll change it to dynamic evaluation